### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alexswiontek**
Would still like some assistance from @thebearingedge

### CODE REVIEW

**Comment by thebearingedge** [comment]
I'll review this as soon as I have a moment.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,86 @@
...
\ No newline at end of file
+## Goals
+* Projects to collaborate on : TBA
+* Goals: Let's try to get at least one commit a day
+* Download: [TeamViewer11](http://teamviewer.com/en/), [Git](https://git-scm.com/downloads)
+
+## Getting Started with Git/GitHub
+To make sure we can all contribute to this repository, make sure to download Git. There is a good guide to setting up found here: [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Once you have Git installed, you can run the following command to ensure it is installed:
+```
+git --version
```
Comment (dinhjeffrey) [inline README.md:17]: hey, I'm wondering how did you format it to look like this?

Comment (dinhjeffrey) [inline README.md:17]: for git --version and cd Downloads

Comment (alexswiontek) [inline README.md:17]: Check out the [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for instructions on how to write markdown. The part about correctly formatting code is [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).

File: README.md [hunk 2]
```diff
@@ -1,7 +1,86 @@
...
+git commit -m "my commit"
+```
+Notice that we can change the contents of the commit message in the quotes ("my commit"). To push changes to your GitHub repository, type in:
+```
+git push origin master
+```
+What this does is push the 'master' branch to the 'origin' remote, in which case should only be ours. See more below on remotes. What has this change done? If everything was done correctly, our forked repository should have "my commit" in the commit history. How do we merge our changes into the main FreeCodeCampOC folder? We can do so by initiating a pull request.
+
+![Pull request](http://i.imgur.com/ZKndI69.png)
+
```
Comment (dinhjeffrey) [inline README.md:67]: do we have to remove a remote after cloning and re-add it somewhere here? @alexswiontek @thebearingedge

**Pull request merged**